exporter/jaegerexporter/exporter.go

```diff
@@ -80,7 +80,7 @@ func (s *protoGRPCSender) pushTraceData(
 	for _, batch := range batches {
 		_, err = s.client.PostSpans(
 			ctx,
-			&jaegerproto.PostSpansRequest{Batch: *batch})
+			&jaegerproto.PostSpansRequest{Batch: *batch}, grpc.WaitForReady(true))
```

Comment:
This has to be configurable otherwise tests are failing. 

Any objections on adding to the main gRPC config?

Reply:
I would like to read the documentation before saying anything. Will do that in 30 mins and confirm